pubtools/exodus/_tasks/push.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import logging
+import os
+import time
+import subprocess
+from six.moves.urllib.parse import urljoin
+
+from pubtools.exodus.task import ExodusTask 
+
+LOG = logging.getLogger("pubtools-exodus")
+LOG_FORMAT = "%(asctime)s [%(levelname)-8s] %(message)s"
+
+
+class ExodusPushTask(ExodusTask):
```

Comment:
we could probably reuse a few methods from https://github.com/release-engineering/pubtools-exodus/blob/main/pubtools/exodus/_hooks/pulp.py

Reply:
Agree, the exodus-gw interactions really should be just implemented and tested once.